server/src/server.js

```diff
@@ -822,31 +824,8 @@ app.get("/contribute.json", function (req, res) {
 });
 
 // FIXME: this can't the right way to do this...
```

Comment:
Great, the non-right way of doing this is no longer the way we do it! Remove the fixme and the next commented line.